ccxcon/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.conf.urls import include, url
+from django.contrib import admin
+
+from rest_framework_nested import routers
+
+from courses.views import CourseViewSet, ModuleViewSet
+
+router = routers.DefaultRouter()
+router.register('coursexs', CourseViewSet)
+
+modules_router = routers.NestedSimpleRouter(router, 'coursexs', lookup='uuid')
```

Comment:
I would also like this test and the other one (maybe rewritten in some form) to be included somewhere so we can guard against this in the future